api/Bluetooth.json

```diff
@@ -203,6 +203,89 @@
           }
         }
       },
+      "getDevices": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Bluetooth/getDevices",
+          "support": {
+            "chrome": {
+              "version_added": "85",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "#enable-experimental-web-platform-features",
+                  "value_to_set": "enabled"
+                }
+              ]
+            },
+            "chrome_android": {
+              "version_added": "85"
```

Comment:
Is this actually not behind a preference on Android, but is behind a preference on desktop?

Reply:
@ddbeck This IS behind a preference on desktop (or at least the ones I tested - chrome, opera, edge).  I _do not know_ whether it is behind a preference on android (links don't say). How do we find out?

Reply:
I ended up [testing this](https://codepen.io/ddbeck/pen/ZEpyvLa?editors=1111) on my phone. In Chrome 87, this is still behind the flag.

Reply:
Thanks! Fixed